src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppElasticBeanstalkWindows.recipe

```diff
@@ -751,6 +752,21 @@
                     "AdvancedSetting": true,
                     "Updatable": false
                 },
+                {
+                    "Id": "CreateNew",
+                    "Name": "Create New VPC",
+                    "Description": "Do you want to create a new VPC?",
+                    "Type": "Bool",
+                    "DefaultValue": false,
+                    "AdvancedSetting": true,
+                    "Updatable": false,
+                    "DependsOn": [
+                        {
+                            "Id": "VPC.UseVPC",
+                            "Value": true
+                        }
+                    ]
+                },
```

Comment:
Not caused here, but while we're working nearby can you remove the "App Runner" reference on line 773 that was likely a copy+paste error?

Reply:
Done